themes/src/main/resources/theme/base/login/messages/messages_en.properties

```diff
@@ -269,11 +269,14 @@ federatedIdentityUnavailableMessage=User {0} authenticated with identity provide
 federatedIdentityUnmatchedEssentialClaimMessage=The ID token issued by the identity provider does not match the configured essential claim. Please contact your administrator.
 
 confirmLinkIdpTitle=Account already exists
+confirmOverrideIdpTitle=Broker link already exists
 federatedIdentityConfirmLinkMessage=User with {0} {1} already exists. How do you want to continue?
+federatedIdentityConfirmOverrideMessage=It seems that the account you are trying to link has already been linked to another account using the identity provider {0}. Can you confirm if you want to replace the existing link with the new account?
```

Comment:
I am not 100% sure if this message is a bit confusing? It is going to be shown to the end user, so perhaps we might come with something more clear, so the user is more clearly aware of what is going on? I can think about something like:
```
You are trying to link your account {0} with the {1} account {2}. But your account is already linked with different {3} account {4}. Can you confirm if you want to replace the existing link with the new account?
```
{0} - Keycloak username
{1} - idp alias
{2} - new federatedIdentity username
{3} - idp alias (same like {1})
{4} - old federatedIdentity username (something like `federatedIdentity.getUsername()` from existing link)

Is it possible? If it is a headache to display existing link, we can possibly skip {4} in the message above (as I am not sure if it requires to add new freemarker bean to display the info about existing link details, but I am not sure if it worth it...).

@pedroigor WDYT?

Reply:
Yeah, your message gives more information and makes clearer what is going on.